### DISCUSSION TITLE
I am trying to make a non-admin user have a view of the dashboard.

### DISCUSSION THREAD

**Discussion by scottgcole511** [snipe/snipe-it#10793] [General]
I am trying to make a non-admin user have a view of the dashboard.

I tried giving a user Admin access so that they can see the dashboard view, but then restricting access to Licenses under permission, I don't want them to see License information. In user permissions, it seems that an Admin overrides and allows all modules to be accessed. Is that correct or am I missing a way/workaround to lock an Admin out of certain modules?

**Comment by yosiasz** [comment]
admin is admin. they will see it all. what specifically is this user's tasks that they would need to see the dashboard?

**Comment by cringerjs** [comment]
I don't think there is currently a way.  I've ran into the same issue with stakeholders wanting to see the dashboard.  Specifically how many computers they have available vs deployed.  Would be nice to have dashboard only view.

**Comment by cryptelli** [comment]
I've recently started using Snipe-IT in my environment and agree that this would be a really good option to provide non admin users with the a visual view of the system as a whole.
